CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # Check CMake version
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
What is in cmake 3.20 that isn't in version 3.10, that is required to build this software?

Reply:
In CMake 3.10, if you use a package that sets a CMake variable equal to <PACKAGE_NAME>_ROOT, the variable will be ignored unless you set CMake policy CMP0074. This requirement was deprecated in 3.2.0.  It is pretty common for modern CMake packages to set variables equal to <PACKAGE_NAME>_ROOT, so I figured I should update the minimum cmake version to 3.20,